utils/cpupower/Makefile

```diff
@@ -0,0 +1,58 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=cpupower
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-2.0-only
+
+# Since kernel 2.6.39.1 userspace tools are inside the kernel tree
+# Package Automatic match version in kernel
+# HASH is not useful kernel package already check it
```

Comment:
We don't need to have it in the Makefile, I suppose. Documentation is elsewhere. :)

Reply:
I modeled this after https://github.com/openwrt/packages/blob/master/net/usbip/Makefile#L15 that contains the comment.  I deleted it.